sdk/python/touca/cli/config.py

```diff
@@ -67,21 +67,11 @@ def run(self):
 touca config set api-key=3c335732-bf44-4b28-9be8-f30c00e7960f
 """
         values: List[str] = self.options.get("key")
-        pairs = [x.split("=", maxsplit=1) for x in values]
+        pairs = [tuple(x.split("=", maxsplit=1)) for x in values]
```

Comment:
Perhaps it no longer makes sense to create `pairs`. Why don't we just create the dict here?